test.sh

```
@@ -0,0 +1,30 @@
+THING_ID="3b0c3d98-a2dd-439b-891b-03aceec88306"
+supermq-cli clients get $THING_ID $DOMAIN_ID $USER_TOKEN
+
+export THING_ID="3b0c3d98-a2dd-439b-891b-03aceec88306"
+export THING_KEY="070067af-b32b-4f4b-9f39-55b609fca2ca"
+export CHANNEL_ID="17126966-3e9c-4d7f-9ec5-c9204701b859"
+
+coap-cli get channels/$CHANNEL_ID/messages -a $THING_KEY -o
+
+mosquitto_sub -u $THING_ID -P $THING_KEY -t channels/$CHANNEL_ID/messages -I supermq -h localhost
+
+ws://localhost:8186/channels/17126966-3e9c-4d7f-9ec5-c9204701b859/messages?authorization=070067af-b32b-4f4b-9f39-55b609fca2ca
```

Shellcheck (0.10.0)

[error] This is interpreted as a command name containing '='. Bad assignment or comparison?

(SC2276)